### DISCUSSION TITLE
Formalize/document a release schedule or cycle

### DISCUSSION THREAD

**Discussion by worldofpeace** [elementary/os#767] [Q&A]
Currently I'm not sure if there's any formal documentation on how exactly `elementary` measures and performs their release process.

I am aware that active development is synced with Ubuntu LTS releases, but other than that my awareness is formed completely from external observation.

**Comment by sumitsaurabh927** [comment]
I'm new to FOSS development and hence I want to start with documentation. Would love to help though. 
How do I go about doing it?

**Comment by cassidyjames** [comment]
For this sort of information, the [OS wiki](https://github.com/elementary/os/wiki) would probably be appropriate. [This wiki page](https://github.com/elementary/os/wiki/Release-Process) currently covers the steps for releasing apps, but doesn't cover scheduling.

**Comment by sumitsaurabh927** [comment]
I checked the wiki and as you mentioned it only has release steps for apps. I'd be able to document the same for the OS if you could point me towards the OS release cycle.

**Comment by cassidyjames** [comment]
I guess the "problem" is we don't have a strict/formal release process. For major versions of the OS itself, we base off of the Ubuntu LTS and thus release some time after each LTS release, around every two years. However, we also have a "when it's ready" policy where we don't set a specific date until we're nearing a featureful release.

For other components, we aim to support a monthly release cycle. Development happens constantly, but near the end of the month we try to make a release for each component that has been updated (if it has not already had a recent release). We document the entirety of releases that happened within the past month at the beginning of the next month (sounds confusing, but i.e. [on May 1 we wrote about all of the updates that happened in April](https://blog.elementary.io/hera-updates-for-april-2020/)). We have begun releasing patch versions of the OS (i.e. 5.1.3, 5.1.4, etc.) along with a new build of the ISO at the beginning of the month, as well.
